time_difference.gemspec

```diff
@@ -11,10 +11,10 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = "time_difference"
   gem.require_paths = ["lib"]
-  gem.version       = "0.7.0"
+  gem.version       = "1.0.0"
```

Comment:
This change is not relevant and should be done in separated PR/commit.

Reply:
Isn't it a breaking change though? I guess I should either keep the gem version or the activesupport version the same. If I bump up active_support to be 6.0, that's a breaking change for people who still want to remain on rails 5.2.

Reply:
Yes, but for alpha versions (`0.y.z`) there are breaking changes in minor version, not major. `1.y.z` is stable release, it's pretty big move. Please, read https://semver.org/

Reply:
"MINOR version when you add functionality in a backwards compatible manner"

This wouldn't be backwards compatible for people who wanted to stay on rails 5.2 and not upgrade to rails 6. Upgrading from rails 5 to 6 is a pretty huge move.

Regardless, we could specify the `activesupport dependency` as `>= 5.1`, then bump up the minor version and still keep rubies 2.2 to 2.4 in the travis build config.

Reply:
I meant this: https://semver.org/#spec-item-4

`0` major version is not regular major version that should be increased with any backwards incompatible change.

Reply:
The latest commit to hit master was almost 2 years ago! I think the api is pretty stable now. I guess this gem could be kept to 0.x ad infinitum. 

Anyway, would specifying the `activesupport` as `>= 5.1` and keeping the gem version to `0.x` satisfy you?

Reply:
> The latest commit to hit master was almost 2 years ago! I think the api is pretty stable now. I guess this gem could be kept to 0.x ad infinitum.

Okay, but anyway, I believe that version should be changed in separated commit (with tag) by maintainers.

> Anyway, would specifying the activesupport as >= 5.1 and keeping the gem version to 0.x satisfy you?

This is good idea while the gem works with lower versions of dependencies.